q2_types/feature_table/_transformer.py

```diff
@@ -106,3 +106,8 @@ def _6(data: biom.Table) -> BIOMV210Format:
     with ff.open() as fh:
         data.to_hdf5(fh, generated_by=_get_generated_by())
     return ff
+
+
+@plugin.register_transformer
+def _7(data: biom.Table) -> pd.DataFrame:
+    return _table_to_dataframe(data)
```

Comment:
Wouldn't it be possible to just decorate `_table_to_dataframe`. It seems to have the annotations already (not sure why), and `register_transformer` returns the transformer unmarred, so it could still be used as is.

That being said, no idea if we want to do that in practice (composing transformers via nesting). @jairideout what do you think?